docs/guides/tor/performance-issues.md

```diff
@@ -39,7 +39,7 @@ This is not recommended, but here are some things you can do:
 
 ##### Solution 2 - Only use specific Exit Nodes
 
-1. Navigate to [atlas.torproject.org Top Relays](https://atlas.torproject.org/#toprelays).
+1. Navigate to [metrics.torproject.org Top Relays](https://metrics.torproject.org/rs.html#toprelays).
```

Comment:
I need a confirmation that the instructions below are still valid since I don't use Tor myself.

Reply:
I don't use TOR myself. The changes are okay, if they are wrong someone will helpfully notify us.